dl4j-cv-labs/src/main/resources/config.properties

```diff
@@ -8,6 +8,8 @@ dataset.dogbreed.url=https://docs.google.com/uc?export=download&id=17dOxbD7Tr3_w
 dataset.segmentationCell.url=https://drive.google.com/a/skymind.my/uc?authuser=0&id=1zHn593J13dxLO1AJ0N2jKhpahs0yYGa0&export=download
 #fruits dataset link
 dataset.fruits.url=https://drive.google.com/a/skymind.my/uc?authuser=0&id=1_d2BgWjsDu24UGWqBzNy52jDxy2zzxrS&export=download
+#metal defects dataset link
+dataset.metaldefects.url=https://www22.zippyshare.com/d/3nRRGgnL/18019/metal-defects.zip
```

Comment:
can stick to drive if zippyshare not working as expected? easier for us to migrate it later on.

Reply:
migrate to the current proposed server